src/dist/replication/common/fs_manager.cpp

```diff
@@ -80,38 +81,39 @@ void dir_node::update_disk_stat()
         disk_capacity_mb = info.capacity / 1024 / 1024;
         disk_available_mb = info.available / 1024 / 1024;
         disk_available_ratio =
-            disk_capacity_mb == 0 ? 0 : disk_available_mb * 100 / disk_capacity_mb;
-        ddebug("update disk space succeed: dir = %s, capacity_mb = %" PRId64
-               ", available_mb = %" PRId64 ", available_ratio = %" PRId64 "%%",
-               full_dir.c_str(),
-               disk_capacity_mb,
-               disk_available_mb,
-               disk_available_ratio);
+            static_cast<int>(disk_capacity_mb == 0 ? 0 : std::round((double)disk_available_mb *
+                                                                    100 / disk_capacity_mb));
+        ddebug_f("update disk space succeed: dir = {}, capacity_mb = {}, available_mb = {}, "
+                 "available_ratio = {}%",
+                 full_dir.c_str(),
+                 disk_capacity_mb,
+                 disk_available_mb,
+                 disk_available_ratio);
     } else {
-        derror("update disk space failed: dir = %s", full_dir.c_str());
+        derror_f("update disk space failed: dir = {}", full_dir.c_str());
     }
 }
 
 fs_manager::fs_manager(bool for_test)
 {
     if (!for_test) {
-        _counter_capacity_total_mb.init_app_counter("eon.replica_stub",
+        _counter_total_capacity_mb.init_app_counter("eon.replica_stub",
```

Comment:
why change this name? you make this PR change too bloated. if you do feel this name is not appropriate, make this rename another PR.

Reply:
acelyc111 suggest modify